java/spring-framework/config.yaml

```diff
@@ -7,9 +7,9 @@ build:
 
 binaries:
   - target/benchmark-1.0.0.war
-  - target/benchmark-1.0.0/WEB-INF/lib/jetty-runner-9.4.34.v20201102.jar
+  - target/benchmark-1.0.0/WEB-INF/lib/jetty-runner-9.4.35.v20201120.jar
```

Comment:
can `maven` export this dependency in a specific file (with a static name) @lukaszlenart ?

Reply:
Looks like it should be possible
https://stackoverflow.com/questions/4969313/renaming-maven-dependency-in-wars-web-inf-lib-folder